docs/developer-docs/integrations/sns/sns-intro.md

```diff
@@ -239,14 +148,14 @@ The SNS communities are responsible for individually topping up the cycles of
 all SNS canisters as well as all dapp canisters that are controlled by the SNS.
 Special care must be taken that cycles are also monitored for canisters that
 are automatically created. In particular, this includes the archive canisters
-that are spawn by the ledger canister.
+that are automatically spawn by the ledger canister.
 **If the archive canisters are not provided
 with sufficient cycles, the ledger block history may be lost.**
 :::
 In the future, cycles management will be simplified in a new feature that 
 allows canister groups, where cycles can be managed across different canisters.
 
-There are separate pages where you can fine more details regarding [how 
+There are separate pages where you can find more details regarding [how 
```

Comment:
TODO: check in deployment preview if links that span multiple lines work

Reply:
typo is not here anymore. 
Noted to check

Reply:
@LaraAS @sesi200 Links seem fine: https://deploy-preview-560--icportal.netlify.app/docs/current/developer-docs/integrations/sns/sns-intro/

Reply:
Thanks for checking. Looks good indeed!